labextension/src/index.ts

```diff
@@ -30,10 +30,10 @@ const TOUR_TIMEOUT = 5 * TOUR_DELAY + 1;
 
 async function activateCondaEnv(
   app: JupyterFrontEnd,
-  palette: ICommandPalette,
-  menu: IMainMenu,
-  restorer: ILayoutRestorer,
-  settingsRegistry: ISettingRegistry
+  settingsRegistry: ISettingRegistry,
```

Comment:
It looks like the settings registry could also be made optional (and have `CondaEnvironments` use a set of defaults)

But this would be fine to do in a separate PR.

Reply:
The settings registry is actually optional here:

https://github.com/fcollonval/jupyter_conda/blob/a7fd182a6b6a47a7ca48b545ba946e2937fcf4b1/labextension/src/services.ts#L61

So we could indeed make it optional at the plugin level too.